.github/workflows/ci.yaml

```
@@ -184,7 +184,8 @@ jobs:
           cache: yarn
       - run: corepack enable yarn
       - run: SKIP_BUILD_INTERNAL_DEPENDENCIES=true yarn install
-      - run: yarn workspaces foreach --recursive --parallel --topological --from @tupaia/${{ matrix.package }} run build
+      - name: Build subtree
+        run: yarn workspaces foreach --recursive --parallel --topological --verbose --jobs unlimited --from @tupaia/${{ matrix.package }} run build
```

Bit unwieldy, but arguably more readable? Could easily be shortened

Feel free to commit a suggestion directly from here

```suggestion
        run: yarn workspaces foreach -Rptv --jobs unlimited --from @tupaia/${{ matrix.package }} run build
```

…or…

```suggestion
        run: yarn workspaces foreach \
          --recursive \
          --parallel \
          --topological \
          --verbose \
          --jobs unlimited \
          --from @tupaia/${{ matrix.package }} \
          run build
```

---

shortened -rptv combination definitely seems more readable to me <3